src/envo/envo-edit.owl

```diff
@@ -41092,7 +41092,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501187> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501188> (ophthalmology clinic)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501188> "A medical clinic which is used to provide opthalmology services."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501188> "A medical clinic which is used to provide opthalmology services."@en)
```

Comment:
@ddooley Is "opthalmology" spelled correctly? Autocorrect suggests "ophthalmology".

Reply:
Good catch!